rules/library.bzl

```diff
@@ -640,11 +641,6 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
 
     for vendored_static_library in kwargs.pop("vendored_static_libraries", []):
         import_name = "%s-%s-library-import" % (name, paths.basename(vendored_static_library))
-        native.objc_import(
-            name = import_name,
-            archives = [vendored_static_library],
-            tags = _MANUAL,
-        )
         vendored_deps.append(import_name)
```

Comment:
Should you move this line so it stays with `native.objc_import`? It'll work at runtime just fine, but I'm worried if someone adds conditionals around the `objc_import` in the future a bad dep would be added.

Reply:
Totally, good catch

Reply:
Fixed!